ci/properties/gradle-publish.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Publish Java Package with Gradle",
```

Comment:
this is rather a large name, please make sure it fits on the card without wrapping

Reply:
Double checked locally, looks good. Even a little wiggle room is left

![image](https://user-images.githubusercontent.com/16109154/75057417-f3cfaf00-54a6-11ea-86e9-51a665afd939.png)